.idea/jsLibraryMappings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same for all the other ones. Also, generally you'd do one PR for issue and specify the issues solved in the PR, and try to limit it to one issue per PR unless they're really closely related